### ISSUE TITLE
Auto-number workspaces in title bar

### ISSUE THREAD

**Issue by scottopell** [rickbutton/workspacer#10]
Currently if I rename a workspace to something non-numeric, eg `web`, then I lose the number. It would be great if it could be displayed as `{2} web`.

This could be the default behavior or something I can configure, doesn't matter.

**Comment by rickbutton** [comment]
I am breaking out the display name of a workspace in the widget into a separate function that can be overridden in your own sublcass of WorkspaceWidget. For example:

```csharp
public class MyWorkspaceWidget : WorkspaceWidget {
    public override string GetDisplayName(IWorkspace workspace, int index, bool visible) {
        return visible ? $"[{{{index}}} {workspace.Name}]" : $" {{{index}}} {workspace.Name} ";
    }
}
```

then you can use MyWorkspaceWidget instead of the default in your config.

this is implemented in 28a23787e568b0fb68e585eda0a68445062acef5, most likely a part of v0.5

**Comment by scottopell** [comment]
I get the following error message:
```
---------------------------
unhandled exception!
---------------------------
System.Exception: (13,35): error CS0506: 'MyWorkspaceWidget.GetDisplayName(IWorkspace, int, bool)': cannot override inherited member 'WorkspaceWidget.GetDisplayName(IWorkspace, int, bool)' because it is not marked virtual, abstract, or override

   at Workspacer.ConfigLoader.ConfigHelper.CompileConfig(IEnumerable`1 referenceTypes)

   at Workspacer.ConfigLoader.ConfigHelper.GetConfig(IEnumerable`1 referenceTypes)

   at Workspacer.Workspacer.Start()

   at Workspacer.Program.Run()

   at Workspacer.Program.Main(String[] args)

press ctrl-c to copy this
---------------------------
OK   
---------------------------

```
